lci_strategic_plugin/src/lci_strategic_plugin_algo.cpp

```diff
@@ -251,6 +250,17 @@ rclcpp::Duration LCIStrategicPlugin::get_earliest_entry_time(double remaining_di
 
 }
 
+int LCIStrategicPlugin::getClearanceDuration(lanelet::CarmaTrafficSignalPtr traffic_light)
```

Comment:
The name is little misleading as it is getting movement allowed durations not clearance